### ISSUE TITLE
Select posts by 'last 7 days' period,  'last 30 days' 

### ISSUE THREAD

**Issue by microtag** [bootsz/wp-advanced-search#200]
I would like to give my users the flexibility of selecting posts for 'last 7 days' period,  'last 30 days' period or "all".
Is possible?
Thanks

**Comment by a42** [comment]
I don't think it is a build in functionality in wpas but with a little code in your functions.php you can achieve this...

Take a look a the date_query used in conjunction with the WP_Query you could get lucky, I guess.

https://codex.wordpress.org/Class_Reference/WP_Query#Date_Parameters

The way to order the search result is done with the orderby-type >> http://wpadvancedsearch.com/param/orderby_values/

/Adam
